<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Problem Magazyniera</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Podaj dane towarów:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>522</width>
             <height>462</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Towar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Czas Jednostkowy</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Częstotliwość</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout1">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Towar 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox1"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox1"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Towar 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox2"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout3">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Towar 3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox3"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox3"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout4">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Towar 4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox4"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox4"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Towar 5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox5"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox5"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Towar 6</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox6"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox6"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout7">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Towar 7</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox7"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox7"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout8">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Towar 8</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox8"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox8"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout9">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Towar 9</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox9"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox9"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout10">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Towar 10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox10"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox10"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout11">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Towar 11</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox11"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox11"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout12">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Towar 12</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox12"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox12"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout13">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Towar 13</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox13"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox13"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout14">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Towar 14</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox14"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox14"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="inputLayout15">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Towar 15</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="timeSpinBox15"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="freqSpinBox15"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Liczba pokoleń</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="generationsNoSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="mutationSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mutationLabel">
            <property name="text">
             <string>0.01</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="getSolutionButton">
            <property name="text">
             <string>Znajdź Rozwiązanie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="outputPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>5</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>6</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>7</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>8</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>9</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>10</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>11</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>12</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>13</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>14</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>15</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Nr Towaru</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Czas Jednostkowy</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Częstotliwość</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Średni czas załadunku towaru</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="averageTimeLabel">
            <property name="text">
             <string>66.6</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Liczba pokoleń</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="generationsNoLabel">
            <property name="text">
             <string>666</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="newTaskButton">
            <property name="text">
             <string>Wprowadź nowe dane</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loadingPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>128</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="font">
             <font>
              <pointsize>48</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ewolucja w toku</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>128</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAuthors"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAuthors">
   <property name="text">
    <string>Autorzy</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
